### ISSUE TITLE
"follow with Fraidycat" icon is hard to see with a Dark browser theme

### ISSUE THREAD

**Issue by yringot** [kickscondor/fraidycat#193]
* [x] I have checked the [Common Problems](https://github.com/kickscondor/fraidycat/wiki/Common-Problems) page.
* [x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Firefox
 - Version 83.0

**Describe the bug**
The "follow with Fraidycat" icon is hard to read against a dark background, such as when using a dark theme: 
![fraidycat](https://user-images.githubusercontent.com/3494882/101240097-53711100-36ed-11eb-9981-b6a567b5dca0.png)

Probably the easiest way to fix this would be to have the inner parts of the icon being a lighter color.

**Comment by kickscondor** [comment]
Yes - it should be respecting dark themes - but in practice it doesn't work.

I think I need to make the icon similar to the normal icon anyway. It's disorienting to have a completely different icon.

**Comment by LeBaux** [comment]
> I think I need to make the icon similar to the normal icon anyway. It's disorienting to have a completely different icon.

You should consider just using 2 variations of current cat icon. 

1. When a user is on a page that has a content that could be added to fraidycat, the tail should be up.
2. Otherwise, the cat should be less excited about the page it is visiting, hence the tail should be down.
